### PR TITLE
epics-pypdb recipe

### PR DESCRIPTION
**Pull request by hhslepicka**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by hhslepicka** [comment]
@conda-forge/help-python I have no idea why this build on windows is failing.
I tested it locally at a Windows box and everything went fine without issues.

**Comment by scopatz** [comment]
Noarch only needs to pass on linux.  This is fine

File: recipes/epics-pypdb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
```
Comment (scopatz) [inline recipes/epics-pypdb/meta.yaml]: you don't need pip and setuptools. Please remove setuptools

Comment (hhslepicka) [reply recipes/epics-pypdb/meta.yaml]: Thank you, @scopatz for the review.
This was addressed with the latest commit.

**Comment by hhslepicka** [comment]
@scopatz this is ready for a new review/merge. Thank you for taking a look at it.

**Pull request merged**